src/plist.c

```diff
@@ -2542,65 +2542,65 @@ static Obj FuncIsRectangularTablePlist(Obj self, Obj plist)
+  { T_PLIST_DENSE_NHOM_SSORT +IMMUTABLE,    "non-homogeneous dense strictly-sorted immutable plain list" },
 
-  { T_PLIST_DENSE_NHOM_NSORT,               "list (plain,dense,nhom,nsort)" },
-  { T_PLIST_DENSE_NHOM_NSORT +IMMUTABLE,    "list (plain,dense,nhom,nsort,imm)" },
+  { T_PLIST_DENSE_NHOM_NSORT,               "non-homogeneous dense non-strictly-sorted plain list" },
+  { T_PLIST_DENSE_NHOM_NSORT +IMMUTABLE,    "non-homogeneous dense non-strictly-sorted immutable plain list" },
 
-  { T_PLIST_EMPTY,                          "list (plain,empty)" },
-  { T_PLIST_EMPTY      +IMMUTABLE,          "list (plain,empty,imm)" },
+  { T_PLIST_EMPTY,                          "empty plain list" },
+  { T_PLIST_EMPTY      +IMMUTABLE,          "immutable empty plain list" },
 
-  { T_PLIST_HOM,                            "list (plain,hom)" },
-  { T_PLIST_HOM        +IMMUTABLE,          "list (plain,hom,imm)" },
+  { T_PLIST_HOM,                            "homogeneous plain list" },
+  { T_PLIST_HOM        +IMMUTABLE,          "homogeneous immutable plain list" },
 
-  { T_PLIST_HOM_NSORT,                      "list (plain,hom,nsort)" },
-  { T_PLIST_HOM_NSORT  +IMMUTABLE,          "list (plain,hom,nsort,imm)" },
+  { T_PLIST_HOM_NSORT,                      "non-strictly-sorted homogeneous plain list" },
+  { T_PLIST_HOM_NSORT  +IMMUTABLE,          "non-strictly-sorted homogeneous immutable plain list" },
 
-  { T_PLIST_HOM_SSORT,                      "list (plain,hom,ssort)" },
-  { T_PLIST_HOM_SSORT +IMMUTABLE,           "list (plain,hom,ssort,imm)" },
+  { T_PLIST_HOM_SSORT,                      "homogeneous strictly-sorted plain list" },
+  { T_PLIST_HOM_SSORT +IMMUTABLE,           "homogeneous strictly-sorted immutable plain list" },
 
-  { T_PLIST_TAB,                            "list (plain,table)" },
-  { T_PLIST_TAB       +IMMUTABLE,           "list (plain,table,imm)" },
+  { T_PLIST_TAB,                            "plain list (table)" },
+  { T_PLIST_TAB       +IMMUTABLE,           "immutable plain list (table)" },
 
-  { T_PLIST_TAB_NSORT,                      "list (plain,table,nsort)" },
-  { T_PLIST_TAB_NSORT +IMMUTABLE,           "list (plain,table,nsort,imm)" },
+  { T_PLIST_TAB_NSORT,                      "non-strictly-sorted plain list (table)" },
+  { T_PLIST_TAB_NSORT +IMMUTABLE,           "non-strictly-sorted immutable plain list (table)" },
 
-  { T_PLIST_TAB_SSORT,                      "list (plain,table,ssort)" },
-  { T_PLIST_TAB_SSORT +IMMUTABLE,           "list (plain,table,ssort,imm)" },
+  { T_PLIST_TAB_SSORT,                      "strictly-sorted plain list (table)" },
+  { T_PLIST_TAB_SSORT +IMMUTABLE,           "strictly-sorted immutable plain list (table)" },
 
-  { T_PLIST_TAB_RECT,                       "list (plain,rect table)" },
-  { T_PLIST_TAB_RECT       +IMMUTABLE,      "list (plain,rect table,imm)" },
+  { T_PLIST_TAB_RECT,                       "plain list (rect table)" },
```

Comment:
What are anyone's thoughts about even expanding `rect` to `rectangular`? Since everything else is being unabbreviated, perhaps this should be too. I think I'd vote for that anyway.

Reply:
I was thinking about the table and rect cases, wasn't sure how to handle them most nicely. They should be adjusted however.